fs-file-management/fs-file-management.md

```diff
@@ -0,0 +1,238 @@
+# Automate file management in R with the {fs} package
+Jadey Ryan
+2024-09-24
+
+Tired of messy folders filled with inconsistently named files? The [`fs`
```

Comment:
Agree! I used my exact use case for when I went through this process at my work. Hopefully this isn't too detailed. Otherwise, we can go with your suggestion. :)